kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/predicates/TestInToFromString.java

```diff
@@ -76,15 +76,28 @@ public void testMultipleStringValues() {
   @Test
   public void testMultipleStringValuesWithEmpty() {
     Assert.assertEquals("In<Utf8>#toString(STRING)",
-        "a,b,",
+        ",a,b",
         Predicates.in(new Utf8("a"), new Utf8("b"), new Utf8("")).toString(STRING));
     Assert.assertEquals("In#toString(STRING)",
-        "a,b,", Predicates.in("a", "b", "").toString(STRING));
+        ",a,b", Predicates.in("a", "b", "").toString(STRING));
```

Comment:
I don't think these should be changed. The `In` implementation should generally preserve order unless normalized to produce a `String`. Normalization should probably be a peer of `toString(Schema)`, `toNormalizedString(Schema)`.